recipe/meta.yaml

```diff
@@ -30,12 +37,122 @@ requirements:
     - make
```

Comment:
it helps to remove these requirements. There is some strange behavior that happens when you have too much shadowing.

Reply:
Remove the whole top level requirements sections